released/packages/coq-coinduction/coq-coinduction.1.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "damien.pous@ens-lyon.fr"
+version: "1.0"
```

Comment:
```suggestion
```

Reply:
we are no longer supposed to indicate the version in the opam file?

Reply:
The version is already defined by the directory path (`coq-coinduction.1.0`). We avoid redundant version specification to make it easier to copy `opam` files.